mLRS/CommonTx/cli.h

```diff
@@ -543,22 +557,29 @@ void tTxCli::print_param(uint8_t idx)
 }
 
 
-void tTxCli::print_param_list(uint8_t flag)
+void tTxCli::print_param_list(void)
 {
-    if ((flag == 0 || flag == 1 || flag == 3) && !connected()) {
+    output_pending = CLI_OUTPUT_PL;
+    if (output_index == 0 && (output_flag == 0 || output_flag == 1 || output_flag == 3) && !connected()) {
         putsn("warn: receiver not connected");
     }
 
-    for (uint8_t idx = 0; idx < SETUP_PARAMETER_NUM; idx++) {
-        if ((flag == 1) && (setup_param_is_tx(idx) || setup_param_is_rx(idx))) continue;
-        if ((flag == 2) && !setup_param_is_tx(idx)) continue;
-        if ((flag == 3) && !setup_param_is_rx(idx)) continue;
+    for (uint8_t count = 0; output_index < SETUP_PARAMETER_NUM; output_index++) {
```

Comment:
no, pl don't do this kind of K&R style C weiredness, either a clear plain for or a clear plain while

Reply:
OK.  Sometimes I think your code is weird too. :)